packages/backend-for-frontend/src/routers/emailDeeplinkRouter.ts

```diff
@@ -0,0 +1,47 @@
+import { ZodiosEndpointDefinitions } from "@zodios/core";
+import { ZodiosRouter } from "@zodios/express";
+import {
+  ExpressContext,
+  ZodiosContext,
+  zodiosValidationErrorToApiProblem,
+} from "pagopa-interop-commons";
+import { bffApi } from "pagopa-interop-api-clients";
+import { emptyErrorMapper, NotificationType } from "pagopa-interop-models";
+import { fromBffAppContext } from "../utilities/context.js";
+import { makeApiProblem } from "../model/errors.js";
+import { config } from "../config/config.js";
+import { notificationTypeToUiSection } from "../model/modelMappingUtils.js";
+
+const emailDeeplinkRouter = (
+  ctx: ZodiosContext
+): ZodiosRouter<ZodiosEndpointDefinitions, ExpressContext> => {
+  const emailDeeplinkRouter = ctx.router(bffApi.emailDeepLinkApi.api, {
+    validationErrorHandler: zodiosValidationErrorToApiProblem,
+  });
+  emailDeeplinkRouter.get(
+    "/emailDeepLink/:notificationType/:entityId",
+    async (req, res) => {
+      const ctx = fromBffAppContext(req.ctx, req.headers);
+
+      const notificationType = NotificationType.parse(
+        req.params.notificationType
+      );
+      try {
+        const url = `${config.frontendBaseUrl}${notificationTypeToUiSection[notificationType]}/${req.params.entityId}`;
```

Comment:
What do you think about this approach?

```node
const url = new URL(
  `${notificationTypeToUiSection[notificationType]}/${req.params.entityId}`,
  config.frontendBaseUrl
).href;
```

This approach correctly handles cases where `config.frontendBaseUrl` is missing the final slash, which is a common misconfiguration